.travis.yml

```diff
@@ -116,6 +114,18 @@ jobs:
         - echo "Building vercel/pkg binary"
         - node build-tools/replace_package_type.cjs module commonjs && npm run build-vercel-pkg
 
+        - echo "version=${TRAVIS_TAG}" > metadata.txt
```

Comment:
Moving this to a script might be better. This file already growing too big.

Reply:
do we have any metadata.txt already